README.md

```diff
@@ -1,5 +1,7 @@
 # autoware_launch
 
-This is one of the prototype repositories for Autoware Core/Universe that AWF agreed to create in the [TSC meeting on 2021/11/17](https://discourse.ros.org/t/technical-steering-committee-tsc-meeting-36-2021-11-17-minutes/23168).
+Launch configuration repository containing node configurations and their parameters.
 
-Please see [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) for more details.
+This meta-repository that constructs an Autoware workspace is [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware).
```

Comment:
On reflection this line looks a little odd. Could we perhaps hyperlink `Autoware` in line 3 to point at `autowarefoundation/autoware` and then remove this line entirely? @kenji-miyake What do you think?

Reply:
You mean like this?
```
# autoware_launch

Launch configuration repository containing node configurations and their parameters.

For Autoware's general repository, see [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware).
```

Reply:
IMO, the word 'meta-repository" is difficult to understand for first-time users. but we want them to lead  `autowarefoundation/autoware` which `autoware_launch` is derived from.

Also, I don't think it matters either way if there is a autoware documentation link or not.

Reply:
More like this?

> # autoware_launch
> 
> A launch configuration repository for [Autoware](https://github.com/autowarefoundation/autoware), containing node configurations and their parameters.

Since the main Autoware repo has an explanation of the main repositories and a link to the documentation site as well, I think that would be sufficient for now.

Reply:
I totally agree! It is the simplest. 
Fixed by https://github.com/autowarefoundation/autoware_launch/pull/47/commits/e9741472454ee3597885a49ffaf71856c52aea06